boards/FSM/Inc/App/App_FsmWorld.h

```diff
@@ -58,16 +60,20 @@ struct HeartbeatMonitor *
  * @param world The world to get primary flow meter for
  * @return The primary flow meter for the given world
  */
-struct FlowMeter *
+struct InRangeCheck *
     App_FsmWorld_GetPrimaryFlowMeter(const struct FsmWorld *world);
 
 /**
  * Get the secondary flow meter for the given world
  * @param world The world to get secondary flow meter for
  * @return The secondary flow meter for the given world
  */
-struct FlowMeter *
+struct InRangeCheck *
     App_FsmWorld_GetSecondaryFlowMeter(const struct FsmWorld *world);
+struct InRangeCheck *
+    App_FsmWorld_GetLeftWheelSpeedSensor(const struct FsmWorld *world);
+struct InRangeCheck *
+    App_FsmWorld_GetRightWheelSpeedSensor(const struct FsmWorld *world);
```

Comment:
Names should be changed to InRangeCheck
```suggestion
    App_FsmWorld_GetSecondaryFlowRateInRangeCheck(const struct FsmWorld *world);
struct InRangeCheck *
    App_FsmWorld_GetLeftWheelSpeedInRangeCheck(const struct FsmWorld *world);
struct InRangeCheck *
    App_FsmWorld_GetRightWheelSpeedInRangeCheck(const struct FsmWorld *world);
```